### DISCUSSION TITLE
Add a mode where you have to retype the current word if you make a mistake

### DISCUSSION THREAD

**Discussion by ghost** [monkeytypegame/monkeytype#1862] [Ideas]
Hello Monkeytype Authors,

It would be nice to have a mode where when you make a mistake in a word, it gets deleted and you have to retype it again, perhaps with a visual effect to notify the typist. The way I think about it, hypothetically it could help make you develop whole word muscle memory more effectively than when you make errors and fix them individually, losing the flow.

Well, what do you think?

TBH, I got that idea from [this reddit post.](https://www.reddit.com/r/typing/comments/kxcmoh/best_way_to_learn_a_new_keyboard_layout/gj9kzoe?utm_source=share&utm_medium=web2x&context=3)

**Comment by unrealapex** [comment]
That would be a good funbox idea.
